client/module/radio.lua

```diff
@@ -234,6 +235,15 @@ RegisterCommand('+radiotalk', function()
 	end
 end, false)
 
+local function setRadioTalkAnim(dict, name)	
+	if dict and name then		
+		radioAnim.dict = dict
+		radioAnim.name = name
+	end	
```

Comment:
There should be a call to `type_check` before this, and we should check if the anim dict actually exists

```suggestion
    type_check({dict, "string"}, {name, "string"})
    if not DoesAnimDictExist(dict) then
      return error(("Dict: %s did not exist"):format(dict))
    end
    radioAnim.dict = dict
    radioAnim.name = name
```